build/views.py

```diff
@@ -27,11 +29,16 @@ def analyze_action(actions, build_obj):
     for node in actions:
         if not node:
             continue
-        if node['_class'] == 'hudson.model.ParametersAction' or node['_class'] == 'com.tikal.jenkins.plugins.multijob.MultiJobParametersAction':
-            build_obj.param = {x['name']: x['value'] for x in node.get('parameters')}
+        print(node['_class'])
```

Comment:
left while debugging?

Reply:
what?

Reply:
I mean that this might need to be cleaned up, unless this was left by intention.